config/locales/en.yml

```diff
@@ -879,6 +879,10 @@ en:
     add_new_sample_type: "Add sample type"
     add_new_sample_group: "Add sample group"
     add_new_column: "Add column"
+    modal_info:
+      head_title: "Information for sample '%{sample}'"
+      title: "This sample is assigned to tasks."
```

Comment:
This one's on me! I know I'm bugging you, but I meant to say "This sample is assigned to N tasks.". (I used `<n>` in the text, and somehow Markdown just hid that part of the text)